src/main/java/uk/gov/hmcts/befta/exception/InvalidPropertyException.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.befta.exception;
+
+import java.io.IOException;
+
+public class InvalidPropertyException extends IOException {
```

Comment:
Please extend this from the base one for functional tests under the same package.

Reply:
I suggest you extend this one, or just use this one instead of having a separate exception type: InvalidTestDataException

Reply:
Also, if you extend from IOException, the catch clause catching this will report a misleading message.

Reply:
I need it to be a subclass of `IOException` so that it will be caught by `JsonTransformer`

Reply:
Resolved in https://github.com/hmcts/befta-fw/pull/123/commits/6d00f97fd256e7a129a7e535d4c70f0928eb64e9